### ISSUE TITLE
Include MNE-NIRS

### ISSUE THREAD

**Issue by hoechenberger** [hoechenberger/mne-installers#14]
MNE-NIRS is now on `conda-forge`, thanks to @rob-luke!!
We should include it in the installer!

**Comment by rob-luke** [comment]
Lets wait till MNE v0.24 as it has some nice NIRS improvements. Do you know an approximate release date? this year?

**Comment by rob-luke** [comment]
We should do the same with connectivity etc. should we have MNE base that just had a minimal install. Then a second mne plus that includes all extra packages in the organisation?

**Comment by hoechenberger** [comment]
Yes so there is already a base package on conda-forge, and I'm planning to build an installer based on that one too :)
